baselines/dom.generated.d.ts

```diff
@@ -14852,6 +14853,59 @@ interface EventSourceInit {
     readonly withCredentials: boolean;
 }
 
+interface AnimationOptions {
+    id?: DOMString;
+    delay?: number;
+    direction?: "normal" | "reverse" | "alternate" | "alternate-reverse";
+    duration?: number;
+    easing?: string;
+    endDelay?: number;
+    fill?: "none" | "forwards" | "backwards" | "both"| "auto";
+    iterationStart?: number;
+}
+
+interface AnimationTimeline {
+    readonly currentTime: number | null;
+}
+
+interface ComputedTimingProperties {
+    endTime: number;
+    activeDuration: number;
+    localTime: number | null;
+    progress: number | null;
+    currentIteration: number | null;
+}
+
+interface AnimationEffectReadOnly {
+    readonly timing: number;
+    getComputedTiming() => ComputedTimingProperties;
+}
+
+interface Animation {
+    currentTime: number | null;
+    effect: AnimationEffectReadOnly;
+    readonly finished: Promise<Animation>;
+    id: string;
+    readonly pending: boolean;
+    readonly playState: "idle" | "running" | "paused" | "finished";
+    playbackRate: number;
+    readonly ready: Promise<Animation>;
+    startTime: number;
+    timeline: AnimationTimeline;
+    oncancel() => void;
```

Comment:
oncancel(): void;

Reply:
missing `onFinish`

Reply:
To be more specific about the event type
`oncancel: (this: Animation, ev: AnimationPlaybackEvent) => any;`
I will add the interface `AnimationPlaybackEvent `